src/views/domain-workflows/domain-workflows-table/domain-workflows-table.tsx

```diff
@@ -87,6 +71,28 @@ export default function DomainWorkflowsTable({ domain, cluster }: Props) {
             isFetchingNextPage={isFetchingNextPage}
           />
         }
+        {...(inputType === 'query'
+          ? {
+              columns: domainWorkflowsTableConfig.map((columnConfig) => ({
```

Comment:
I could keep the columns field above and it would still keep all the columns unsortable if onSort isn't passed, but then it's less obvious why sortable columns are being rendered without the sort toggle.